### PR TITLE
Reference EU Contracts database & P2B impact assessment as reuses

### PR DESCRIPTION
**Pull request by clementbiron**

### CODE REVIEW

File: content/impact/compliance.fr.md [hunk 1]
```diff
@@ -9,7 +9,8 @@ aliases: /fr/domain/compliance
...
 - En mars 2023, l'essai de conformité P2B a suffisamment démontré la valeur d'Open Terms Archive pour que la Commission européenne [l'adopte](https://github.com/OpenTermsArchive/engine/issues/1001) comme outil fondamental pour sa nouvelle équipe interne d'application de l'ASD au sein de son Centre commun de recherche.
-- En novembre 2022, la Commission européenne a utilisé Open Terms Archive pour une [évaluation à grande échelle](https://ppmi.lt/news-insights/ppmi-has-completed-early-evaluation-p2b-regulation) de la conformité au règlement P2B.
+- En novembre 2022, la Commission européenne a [publié une rapport](https://op.europa.eu/en/publication-detail/-/publication/d6a287b5-5116-11ee-9220-01aa75ed71a1/language-en/) utilisant Open Terms Archive pour faire [évaluation à grande échelle](https://ppmi.lt/news-insights/ppmi-has-completed-early-evaluation-p2b-regulation) de la conformité au règlement P2B.
```
Comment (Ndpnt) [inline content/impact/compliance.fr.md:14]:
```suggestion
- En novembre 2022, la Commission européenne a [publié une rapport](https://op.europa.eu/en/publication-detail/-/publication/d6a287b5-5116-11ee-9220-01aa75ed71a1/language-en/) utilisant Open Terms Archive pour faire une [évaluation à grande échelle](https://ppmi.lt/news-insights/ppmi-has-completed-early-evaluation-p2b-regulation) de la conformité au règlement P2B.
```

File: content/impact/compliance.fr.md [hunk 2]
```diff
@@ -9,7 +9,8 @@ aliases: /fr/domain/compliance
 
 ### Réalisations
 
+- Le 1 décembre 2023, la Commission européenne a lancé [officiellement](https://digital-strategy.ec.europa.eu/en/news/commission-launches-new-database-track-digital-services-terms-and-conditions) une nouvelle [base de données](https://platform-contracts.digital-strategy.ec.europa.eu/index.html) qui s'appuie sur Open Terms Archive pour suivre les conditions générales des services numériques.
 - En mars 2023, l'essai de conformité P2B a suffisamment démontré la valeur d'Open Terms Archive pour que la Commission européenne [l'adopte](https://github.com/OpenTermsArchive/engine/issues/1001) comme outil fondamental pour sa nouvelle équipe interne d'application de l'ASD au sein de son Centre commun de recherche.
```
Comment (Ndpnt) [inline content/impact/compliance.fr.md:13]:
```suggestion
- En mars 2023, l'essai de conformité P2B a suffisamment démontré la valeur d'Open Terms Archive pour que la Commission européenne [l'adopte](https://github.com/OpenTermsArchive/engine/issues/1001) comme outil fondamental pour sa nouvelle équipe interne d'application du DSA au sein de son Centre commun de recherche.
```

**Review by MattiSG — CHANGES_REQUESTED** [review]

File: data/reuses/digital-services-terms-and-conditions-database.yaml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+title:
+  en: Digital Services Terms and Conditions Database
+  fr: Base de données des conditions générales des services numériques
+author:
+  en: DGCNECT
+  fr: DGCNECT
+description:
+  en: Database from an European Commission initiative which features the terms and conditions of digital services, with a focus on online platforms such as social media, app stores or marketplaces.
+  fr: Base de données issue d'une initiative de la Commission européenne qui présente les conditions générales des services numériques, en mettant l'accent sur les plateformes en ligne telles que les médias sociaux, les magasins d'applications ou les places de marché.
```
[the remark this answers is not in the document]
Comment (clementbiron) [reply data/reuses/digital-services-terms-and-conditions-database.yaml:9]: I haven't found anything better 🤔

File: content/impact/compliance.md [hunk 4]
```diff
@@ -8,7 +8,7 @@ aliases: /domain/compliance
...
-- In November 2022, the European Commission used Open Terms Archive for a [large-scale assessment](https://ppmi.lt/news-insights/ppmi-has-completed-early-evaluation-p2b-regulation) of compliance with the P2B regulation.
+- On 1 December 2023, the European Commission [officially](https://digital-strategy.ec.europa.eu/en/news/commission-launches-new-database-track-digital-services-terms-and-conditions) launched a new [database](https://platform-contracts.digital-strategy.ec.europa.eu/index.html) based on Open Terms Archive to track the terms and conditions of digital services.
+- In November 2022, the European Commission has [finalised a report](https://op.europa.eu/en/publication-detail/-/publication/d6a287b5-5116-11ee-9220-01aa75ed71a1/language-en/) using the Open Terms Archive to conduct a large-scale [assessment](https://ppmi.lt/news-insights/ppmi-has-completed-early-evaluation-p2b-regulation) of compliance with the [P2B Regulation](https://eur-lex.europa.eu/eli/reg/2019/1150/oj).
```
Comment (MattiSG) [inline content/impact/compliance.md:12]: Please follow our copywriting guidelines.

```suggestion
- In November 2022, the European Commission has [finalised a report](https://op.europa.eu/en/publication-detail/-/publication/d6a287b5-5116-11ee-9220-01aa75ed71a1/language-en/) using Open Terms Archive to conduct a large-scale [assessment](https://ppmi.lt/news-insights/ppmi-has-completed-early-evaluation-p2b-regulation) of compliance with the [P2B Regulation](https://eur-lex.europa.eu/eli/reg/2019/1150/oj).
```

File: data/reuses/p2b-impact-assessment.yaml [hunk 5]
```diff
@@ -0,0 +1,11 @@
+title:
+  en: P2B impact assessment
```
Comment (MattiSG) [inline data/reuses/p2b-impact-assessment.yaml:2]:
```suggestion
  en: P2B regulation impact assessment
```

File: data/reuses/p2b-impact-assessment.yaml [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+  fr: Évaluation de l'impact du réglement européen P2B
```
Comment (MattiSG) [inline data/reuses/p2b-impact-assessment.yaml:3]:
```suggestion
  fr: Étude d'impact du règlement P2B
```

File: data/reuses/p2b-impact-assessment.yaml [hunk 7]
```diff
@@ -0,0 +1,11 @@
...
+author:
+  en: European Commission (DGIM) & Open Evidence
+  fr: Commission européenne (DGIM) & Open Evidence
+description:
+  en: Study on evaluation of the Regulation (EU) 2019/1150 on promoting fairness and transparency for business users of online intermediation services (the P2B Regulation).
+  fr: Étude sur l'évaluation du règlement (UE) 2019/1150 relatif à la promotion de l'équité et de la transparence pour les entreprises utilisatrices de services d'intermédiation en ligne (le règlement P2B).
```
Comment (MattiSG) [inline data/reuses/p2b-impact-assessment.yaml:9]:
```suggestion
  en: Large-scale assessment of compliance with the Regulation (EU) 2019/1150 on promoting fairness and transparency for business users of online intermediation services (“P2B Regulation”).
  fr: Évaluation à grande échelle de conformité au règlement (UE) 2019/1150 relatif à la promotion de l'équité et de la transparence pour les entreprises utilisatrices de services d'intermédiation en ligne (« règlement P2B »).
```

**Review by MattiSG — CHANGES_REQUESTED** [review]

**Comment by MattiSG** [comment]
Can we please make the illustrations full width, like the others? 🙂 

image

**Review by Ndpnt — APPROVED** [review]

**Comment by MattiSG** [comment]
> Nicolas did it
I don't understand 😅

**Pull request merged**